adam-core/src/main/scala/org/bdgenomics/adam/rdd/read/FlagStat.scala

```diff
@@ -93,15 +94,15 @@ object FlagStat {
         val (primaryDuplicates, secondaryDuplicates) = DuplicateMetrics(p)
         new FlagStatMetrics(1,
           primaryDuplicates, secondaryDuplicates,
-          b2i(p.getReadMapped),
-          b2i(p.getReadPaired),
-          b2i(p.getReadPaired && p.getFirstOfPair),
-          b2i(p.getReadPaired && p.getSecondOfPair),
-          b2i(p.getReadPaired && p.getProperPair),
-          b2i(p.getReadPaired && p.getReadMapped && p.getMateMapped),
-          b2i(p.getReadPaired && p.getReadMapped && !p.getMateMapped),
-          b2i(mateMappedToDiffChromosome),
-          b2i(mateMappedToDiffChromosome && p.getMapq >= 5),
+          b2i(b(p.getReadMapped)),
+          b2i(b(p.getReadPaired)),
+          b2i(b(p.getReadPaired) && b(p.getFirstOfPair)),
+          b2i(b(p.getReadPaired) && b(p.getSecondOfPair)),
+          b2i(b(p.getReadPaired) && b(p.getProperPair)),
+          b2i(b(p.getReadPaired) && b(p.getReadMapped) && b(p.getMateMapped)),
+          b2i(b(p.getReadPaired) && b(p.getReadMapped) && b(!p.getMateMapped)),
+          b2i(b(mateMappedToDiffChromosome)),
+          b2i(b(mateMappedToDiffChromosome && p.getMapq >= 5)),
```

Comment:
you mean to guard against a NPE if `p.getMapq` is null? Sure, that makes sense, will do